### DISCUSSION TITLE
Περιγραφή της διαδικασίας της ενδιάμεσης παρουσίασης

### DISCUSSION THREAD

**Discussion by babismacheras** [courses-ionio/sw#1602] [Q&A] [answered]
Θα ήθελα κάποιος που έχει ήδη παρουσιάσει να περιγράψει όσο το δυνατόν αναλυτικότερα το τι περιλαμβάνει μια ενδιάμεση παρουσίαση. Τι και πως παρουσιάζουμε, τι ερωτήσεις μας γίνονται, τι περιλαμβάνει η συζήτηση που αναφέρεται.

**Comment by p20sami** [comment, marked as answer]
Συνοπτικά, παρουσιάζονται τα παραδοτέα της εργασίας με την αντίστοιχη τεκμηρίωση για τις επιλογές, για παράδειγμα για ποιον λογο διαλέξαμε να προσθέσουμε τις τάδε φωτογραφίες σε παραδοτεο του συμμετοχικού περιεχομένου ή πως καταλήξαμε στη διανομή Linux που εγκαταστήσαμε (με ποια κριτήρια δηλαδή, γιατί αυτή τη διανομή και γιατί κάποιο συγκεκριμένο σύστημα init). Οι ερωτησεις ουσιαστικά είναι ερωτησεις τεκμηρίωσης όπως προανέφερα (για ποιον λογο για παράδειγμα αυτή τη διανομή ή γιατί δεν δοκιμάσαμε κάποια άλλη αντίστοιχη διανομή), αλλά υπάρχουν και ερωτησεις που βασίζονται στα βίντεο κουίζ για παράδειγμα αν θέλεις να αναφέρεις κατι που σε ενδιέφερε από τα βίντεο και να γινει συζήτηση πάνω σε αυτό. Γενικά σαν συζήτηση θεωρώ πως είναι όλα όσα ανέφερα και το που μπορει η αναφορά να βελτιωθεί μετά την παρουσίαση για παράδειγμα, ενώ πιστεύω πως μπορούμε να σχολιάσουμε και αναφορές άλλων ατόμων που παρουσιάζουν εκείνη τη στιγμή. Ουσιαστικά αυτά περιέχει η παρουσίαση αν δεν ξέχασα κατι αλλά γενικά κάθε αναφορά και ατομο θα έχει διαφορετικό feedback ανάλογα με το που βρίσκεται, όποτε μην πάρεις σαν απόλυτη την απάντηση αλλά σαν μια ιδέα για το τι υπάρχει στην παρουσίαση. Σίγουρα υπάρχουν και άλλες σκοπιές όποτε σε παροτρύνω να δεις και τις άλλες μελλοντικές απαντήσεις του discussion για να έχεις πλήρη εικονα.

**Reply by babismacheras** [reply]
Ευχαριστώ!
